src/main/java/net/mcreator/java/ClassFinder.java

```diff
@@ -39,46 +39,48 @@
 
 	private static final Logger LOG = LogManager.getLogger("Class Finder");
 
-	public static DeclarationFinder.InClassPosition fqdnToInClassPosition(Workspace workspace, String classfqdn,
+	public static DeclarationFinder.InClassPosition fqdnToInClassPosition(Workspace workspace, String classIn,
 			String packagefqdn, JarManager jarManager) {
 		DeclarationFinder.InClassPosition position = new DeclarationFinder.InClassPosition();
+		String classFQDN;
 
 		// if there is no package, it is a class in the current package
-		if (!classfqdn.contains(".")) {
+		if (!classIn.contains(".")) {

```

Comment:
In previous code, if `!classIn.contains(".")` was false, classfqdn was essentiall why you set in `classFQDN = classIn`.

So I am not sure what change was introduced here. Please explain as I think I am missing something obvious :D

Reply:
It's to fix the issue with java.lang classes: if I clicked on `Object`, it attaches "net.mcreator. ...", so the editor would end up looking for `java.lang.net.mcreator. .Object`, which obviously doesn't exist

Reply:
I see ;)